src/Stratis.Bitcoin/Consensus/ChainedHeaderTree.cs

```diff
@@ -338,7 +342,7 @@ public List<ChainedHeaderBlock> PartialValidationSucceeded(ChainedHeader chained
             if ((chainedHeader.Previous.BlockValidationState != ValidationState.PartiallyValidated) &&
                 (chainedHeader.Previous.BlockValidationState != ValidationState.FullyValidated))
             {
-                this.logger.LogTrace("(-)[PREV_BLOCK_NOT_VALIDATED]:null");
+                this.logger.LogTrace("(-)[PREV_BLOCK_NOT_VALIDATED]:{0}", chainedHeader.Previous);
```

Comment:
I thought our convention was to log the return value of the function in the exit log. Not a big deal, but still.

Reply:
Ok yes I will change :)